docker-compose.prod.yml

```diff
@@ -4,7 +4,7 @@ services:
   web:
     build:
       context: ./app
-      dockerfile: Dockerfile
+      dockerfile: app/Dockerfile
```

Comment:
Did you need to change this to get this docker file to work? The `dockerfile` setting is a relative path from the `context` setting above, and so it should work without `app/`. I'd actually suggest not using the `docker-compose.prod` file locally, and instead just use `docker-compose up`. We can chat about this on our next call if need be.